modules/insights-operator-about.adoc

```diff
@@ -5,10 +5,17 @@
 [id="insights-operator-about_{context}"]
 = About the Insights Operator
 
-The Insights Operator periodically gathers anonymized configuration and component failure status and reports that to Red Hat. This is a subset of the information captured by the `must-gather` tool and allows Red Hat to assess important configuration and deeper failure data than is reported via Telemetry. This data is sent several times a day and describes:
+The Insights Operator periodically gathers configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry.
 
-* Important configuration information about the environment that the cluster runs in
-* Details about the state of the cluster and its major components
-* Debugging information about infrastructure Pods or nodes that are reporting failures
+Users of {product-title} can display the report of each cluster in {cloud-redhat-com}. If any issues have been identified, Insights provides further details and, if available, steps on how to solve a problem.
+
+The Insights Operator does not collect identifying information, such as user names, passwords, or certificates. However, to provide specific remediation steps, the Insights Operator does not anonymize certain information internal to the cluster, such as IP addresses and host names of nodes.
+
+Red Hat uses all connected cluster information to:
+
+* Proactively identify potential cluster issues and provide a solution and preventive actions in {cloud-redhat-com}
+* Improve the {product-title}
```

Comment:
I think we can remove "the" before {product-title}. {product-title} renders as OpenShift Container Platform in our docs.